[DOCS] Removes index pattern from Discover docs (#118811)
elastic/kibana: docs/user/discover.asciidoc

[partintro]
--
You have questions about your data.
What pages on your website contain a
specific word or phrase? What events were logged most recently?
What processes take longer than 500 milliseconds to respond?

With *Discover*, you can quickly gain insight to your data: search and filter your data, get information

> about the structure of the fields, and present your findings in a visualization.

Reviewer: ```suggestion
about the structure of the fields, and display your findings in a visualization.
```

. Select the data you want to work with.
+

> {kib} uses a <<data-views,data view>> to tell it where to find

Reviewer: ```suggestion
{kib} uses a <<data-views,{data-source}>> to tell it where to find
```

your {es} data.

> To view the ecommerce sample data, make sure the data view is set to **kibana_sample_data_ecommerce**.

Reviewer: ```suggestion
To view the ecommerce sample data, make sure the {data-source} is set to **kibana_sample_data_ecommerce**.
```

+
[role="screenshot"]

> image::images/discover-data-view.png[How to set the data view in Discover, width=50%]

Reviewer: ```suggestion
image::images/discover-data-view.png[How to set the {data-source} in Discover, width=50%]
```

. To rearrange the table columns, hover the mouse over a
column header, and then use the move control.

. To view more of the document table, click *Chart options > Hide chart*.

[float]
[[add-field-in-discover]]

> === Add a field to your data view

Reviewer: ```suggestion
=== Add a field to your {data-source}
```

What happens if you forgot to define an important value as a separate field? Or, what if you
want to combine two fields and treat them as one? This is where {ref}/runtime.html[runtime fields] come into play.

> You can add a runtime field to your data view from inside of **Discover**,

Reviewer: ```suggestion
You can add a runtime field to your {data-source} from inside of **Discover**,
```

and then use that field for analysis and visualizations,
the same way you do with other fields.

> . Click the ellipsis icon (...), and then click *Add field to data view*.

Reviewer: ```suggestion
.  Next to the *Index pattern* dropdown, open the action menu (…), and then click *Add field to {data-source}*.
```

+
[role="screenshot"]

> image:images/add-field-to-data-view.png[Dropdown menu located next to data view field with item for adding a field to a data view, width=50%]

Reviewer: ```suggestion
image:images/add-field-to-data-view.png[Dropdown menu located next to {data-source} field with item for adding a field to a {data-source}, width=50%]
```

Save your search so you can repeat it later, generate a CSV report, or use it in visualizations, dashboards, and Canvas workpads.
Saving a search saves the query text, filters,

> and current view of *Discover*&mdash;the columns selected in the document table, the sort order, and the data view.

Reviewer: ```suggestion
and current view of *Discover*&mdash;the columns selected in the document table, the sort order, and the {data-source}.
```
